project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/c4m_user_notifications.module

```diff
@@ -279,13 +313,29 @@ function c4m_user_notifications_form_og_ui_add_users_alter(&$form, &$form_state)
  * Triggers email notification when recommendation flag is set.
  */
 function c4m_user_notifications_flag_flag($flag, $entity_id, $account, $flagging) {
-  if ($flag->name != 'subscribe_c4m_follow_node') {
+  // Don't send notifications during dummy content migration.
+  if (c4m_user_notifications_dummy_content_migration()) {
+    return;
+  }
+
+  if ($flag->name != 'c4m_og_content_recommend') {
     // If not recommendation flag, return early.
     return;
   }
 
-  c4m_user_notifications_notify_author_content_recommended($flag, $entity_id, $account, $flagging);
-  c4m_user_notifications_notify_subscribed_editor_content_recommended($flag, $entity_id, $account, $flagging);
+  if (!og_context()) {
+    // If not at OG context, return early.
+    return;
+  }
+
+  $node = node_load($entity_id);
+  if (!in_array($node->type,
+    array('wiki_page', 'discussion', 'document', 'event'))) {
```

Comment:
@ordavidil 
Coder was complaining.
The line was over 80 characters, and in this case it said that each array value should be in separate line,
I split it into 2 lines.